src/asm3/sitedefs.py

```diff
@@ -281,6 +281,7 @@ def get_dict(k: str, dv: Dict = {}) -> Dict:
 ANIBASE_BASE_URL = get_string("anibase_base_url", "")
 ANIBASE_API_USER = get_string("anibase_api_user", "")
 ANIBASE_API_KEY = get_string("anibase_api_key", "")
+AVID_US_POST_URL = get_string("avid_us_post_url", "https://api.avidid.com/v1/registrations/")
```

Comment:
This should be blank, and this being blank sets hasavid to false in the config.js endpoint like other publishers. The URL should only be in the asm3.conf file rather than the source